python/kserve/kserve/model_server.py

```diff
@@ -204,16 +219,21 @@ def __init__(
             self._grpc_server = GRPCServer(
                 grpc_port, self.dataplane, self.model_repository_extension
             )
-
-        # Logs can be passed as a path to a file or a dictConfig.
-        # We rely on Uvicorn to configure the loggers for us.
-        if configure_logging:
-            self.log_config = (
-                log_config if log_config is not None else KSERVE_LOG_CONFIG
+        self.log_config = log_config
+        if configure_logging is not None:
+            warnings.warn(
+                "The 'configure_logging' argument is deprecated. Instead, Use standard "
+                "function 'configure_logging' from 'kserve.logging' module and explicitly set "
+                "'log_config' arg to 'None' or use --configure_logging cmd arg to disable logger "
+                "configuration."
+            )
+            if not configure_logging:
+                # By setting log_config to None we tell Uvicorn not to configure logging
+                self.log_config = None
```

Comment:
For backwards compatibility, I think we can check if the log handler is already set, we can still configure via uvicorn if the log handler is not provided.

Reply:
Done